### DISCUSSION TITLE
KVCache Testing Setting

### DISCUSSION THREAD

**Discussion by Change72** [ornladios/ADIOS2#4263] [Q&A] [answered]
I encountered an issue with the KVCache testing logic. Unlike adios2_remote_server, which is located in the /bin directory, the current redis-server is built in a separate path.

@eisenhauer @vicentebolea  Do you have any suggestions for how to properly launch the cache server during testing?

**Comment by eisenhauer** [comment, marked as answer]
You might do something like the current PR for XRootD testing.  (That PR needs some tweaking yet, but is basically functional.). In that circumstance, cmake not only looks for the XRootD libraries that it needs to run, but also looks for the XrootD server.  you can make testing conditional upon finding the server.
